wallet/src/main/java/com/android/identity_credential/wallet/navigation/WalletNavigation.kt

```diff
@@ -13,16 +14,44 @@ import com.android.identity_credential.wallet.ReaderModel
 import com.android.identity_credential.wallet.WalletApplication
 import com.android.identity_credential.wallet.ui.destination.about.AboutScreen
 import com.android.identity_credential.wallet.ui.destination.addtowallet.AddToWalletScreen
+import com.android.identity_credential.wallet.ui.destination.document.CredentialInfoScreen
 import com.android.identity_credential.wallet.ui.destination.document.DocumentDetailsScreen
 import com.android.identity_credential.wallet.ui.destination.document.DocumentInfoScreen
-import com.android.identity_credential.wallet.ui.destination.document.CredentialInfoScreen
 import com.android.identity_credential.wallet.ui.destination.document.EventLogScreen
 import com.android.identity_credential.wallet.ui.destination.main.MainScreen
 import com.android.identity_credential.wallet.ui.destination.provisioncredential.ProvisionDocumentScreen
 import com.android.identity_credential.wallet.ui.destination.qrengagement.QrEngagementScreen
 import com.android.identity_credential.wallet.ui.destination.reader.ReaderScreen
 import com.android.identity_credential.wallet.ui.destination.settings.SettingsScreen
 
+/**
+ * Lambda that takes in a NavController and route string and navigates to the corresponding
+ * composable Screen or perform a pop of the back stack.
+ */
+val navigateTo: (NavController, String) -> Unit = { navController, routeWithArgs ->
```

Comment:
great catch - this should be changed as well and it's here because `val navigateTo` used to be declared in `fun WalletNavigation()` (as a lambda) to directly pass arguments from the callback `onNavigate` of `composable` declarations and with the Credential Offer feature we need to navigate before the definition of `WalletNavigation`. I moved `navigateTo` lambda outside of it so it can be referenced globally (rather than being passed down through Compose) and it makes sense that it should be a function rather than a lambda. +1 👍